config/cscs.py

```diff
@@ -1074,6 +1075,13 @@
                 'PrgEnv-pgi'
             ]
         },
+        {
+            'name': 'PrgEnv-nvidia',
+            'modules': [
+                'PrgEnv-nvidia',
+                'cdt/21.05'
```

Comment:
```suggestion
                
```
use `-m cdt/21.05` or `-m cdt-cuda/21.05` on the command line ?

Reply:
@jgphpc If I remove this one, my coda-aware MPI checks fail

Reply:
Can we add a `#FIXME` on top of this `cdt` entry? 
We should handle this differently in the future...